src/devices/Tca955x/samples/README.md

```diff
@@ -0,0 +1,3 @@
+﻿# TODO: This needs to be determined
```

Comment:
just make it simple :-) typically it's the usage you can add like for the main README.md